.gitignore

```diff
@@ -2,6 +2,11 @@
 logs
 *.log
 npm-debug.log*
+warnings.json
+
+# Generated styles
+style.css
+style.css.map
 
```

Comment:
I think that it is good so. Maybe we will later add external CSS libraries which are not on bower/npm.